core/chains/evm/headtracker/head_listener_test.go

```diff
@@ -105,7 +105,7 @@ func Test_HeadListener_NotReceivingHeads(t *testing.T) {
 	evmcfg := evmtest.NewChainScopedConfig(t, cfg)
 	evmcfg.BlockEmissionIdleWarningThreshold()
 	chStop := make(chan struct{})
-	hl := headtracker.NewHeadListener(lggr, ethClient, evmcfg, chStop)
+	hl := headtracker.NewHeadListener(lggr, ethClient, headtracker.NewEvmConfig(evmcfg), chStop)
```

Comment:
If you drop `EVM` from `Config`, then worst case is that callers have to alias the package:
```go
	hl := headtracker.NewHeadListener(lggr, ethClient, evmheadtracker.NewConfig(evmcfg), chStop)
```
But it seems like everything can just do `headtracker.NewConfig`, no?

Reply:
Thanks for the notes - dropping the EVM from config makes sense. I think `NewWrappedConfig` sounds like it is clearer.